Contents/WeaponModification/WeaponModificationPlayer.cs

```diff
@@ -23,7 +24,7 @@ internal class WeaponModificationPlayer : ModPlayer {
 		public float Recharge = 0;
 		public int currentIndex = 0;
 
-		public float damage = 1;
+		public float damage = 0;
```

Comment:
It might be a good idea to change this to `StatModifier`, since what this really does just modifies projectile damage.
It would also allow adding 'flat' values and 'multiply' final value (if `StatModifier` is used instead).